meta/store.go

```diff
@@ -96,6 +96,9 @@ type Store struct {
 	// The amount of time without an apply before sending a heartbeat.
 	CommitTimeout time.Duration
 
+	// Authentication cache.
+	authCache map[string]string
```

Comment:
Technically speaking, I think this also needs synchronization. The code which checks the cache could run in a different goroutine than that which process `DROP